js/clinicalTimeline.js

```diff
@@ -5,7 +5,7 @@ d3 = require('d3');
 var clinicalTimeline = (function(){
   var allData,
     colorCycle = d3.scale.category20(),
-    margin = {left: 200, right:30, top: 15, bottom:0},
+    margin = {left: 200, right:30, top: 15, bottom:0, overviewAxis: {left: 15, right: 15}},
```

Comment:
I don't think this is very clean, because the data structure of `margin` is defined in `d3-timeline`. It's a backwards compatible change, so it's not a big deal, but in general it's better to keep data structures from another library as is. The `overviewAxis` is also not essential for every instance of the clinicalTimeline, so it would be better to somehow move this variable to the plugin. Then try to think of a way how one should pass configuration values to the plugins. Is it the developer including the library that needs to pass the appropriate values to the plugin (in that case that should be done in `index.html`) or are all the plugins called from within `clinicalTimeline` with some default values? What would be a clean case for the latter to pass configuration values to the plugin?